andela/trime
# Fix broken links

### What does this PR do?
Fixes broken links.

### Description of Task to be completed?
Remove links pointing to nowhere.

### How should this be manually tested?
Open the menu bar menu home page. You should only see links for `Start Triming` if you're not logged in or `Profile` and `Logout` if you're logged in.

### What are the relevant pivotal tracker stories?
[Fix all broken links](https://www.pivotaltracker.com/story/show/119618761)
